ckan/config/deployment.ini_tmpl

```diff
@@ -191,6 +191,8 @@ ckan.hide_activity_from_users = %(ckan.site_id)s
 #smtp.password = your_password
 #smtp.mail_from =
 
+## Background Job Settings
+bg_job_timeout = 180
```

Comment:
Yes, namespacing the config options looks tidier. What about

```suggestion
ckan.jobs.timeout = 180
```

Reply:
No problem. Fixing it now.